src/pages/SummaryPage.tsx

```diff
@@ -31,17 +30,18 @@ const TableCell = styled.td`
 
 const SummaryPage = (props: RouteComponentProps) => {
   const { contests, resetBallot, votes } = useContext(BallotContext)
-  const startOver = () => {
+  const getNewBallot = () => {
     if (
       Object.keys(votes).length === 0 ||
       window.confirm('Clear all votes and start over?')
     ) {
       resetBallot()
+      props.history.push('/')
```

Comment:
what's the thinking behind separating the navigation out of `resetBallot()`? Is there a chance we would reset ballot without returning to the home screen and is that bad?

Reply:
👍  Yeah, I noticed this too. I'd like it to be part of the `resetBallot()` function. Trouble is that the `history` prop is only available within the router. But… when we get to the task of replacing the router with a state-based navigation, then we can do this. So, it'll certainly get refactored then. 

I'm gonna refactor the "wip" commits out of this branch and then I'll be ready to merge it. Looking forward to getting into the EMS!

Reply:
👍